templates/patroni.yml.j2

```diff
@@ -140,6 +140,7 @@ postgresql:
   bin_dir: /snap/charmed-postgresql/current/usr/lib/postgresql/{{ version }}/bin
   data_dir: {{ data_path }}
   parameters:
+    shared_preload_libraries: 'timescaledb,pgaudit'
```

Comment:
How does CI tests passed with this line? LGTM.

Reply:
I think the issue affects only the followers, which is why the tests pass. I think going forward we should try to move most (all?) bootstrap params into the regular params, so we have a single source of truth.

Reply:
ACK. the patch should work AS IS, AFAIK. Do you want to merge it or proceed with refactoring now?

Reply:
> ACK. the patch should work AS IS, AFAIK. Do you want to merge it or proceed with refactoring now?

Let's merge this as is and leave the refactoring for better days.